plugins/woocommerce/includes/admin/helper/class-wc-helper-admin.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * WooCommerce Admin Helper - React admin interface
+ *
+ * @package WooCommerce\Admin\Helper
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * WC_Helper Class
+ *
+ * The main entry-point for all things related to the Helper.
+ * The Helper manages the connection between the store and
+ * an account on WooCommerce.com.
+ */
+class WC_Helper_Admin {
+
+	/**
+	 * Loads the class, runs on init
+	 *
+	 * @return void
+	 */
+	public static function load() {
+		add_filter( 'woocommerce_admin_shared_settings', array( __CLASS__, 'add_marketplace_settings' ) );
+	}
+
+	/**
+	 * Pushes settings onto the WooCommerce Admin global settings object (wcSettings).
+	 *
+	 * @param mixed $settings The settings object we're amending.
+	 *
+	 * @return mixed $settings
+	 */
+	public static function add_marketplace_settings( $settings ) {
+		$settings['wccomHelper'] = array(
+			'isConnected' => WC_Helper::is_site_connected(),
+			'connectURL'  => self::get_connection_url(),
+			'userEmail'   => WC_Helper_Options::get( 'auth_user_data' ) ? WC_Helper_Options::get( 'auth_user_data' )['email'] : '',
+			'userAvatar'  => WC_Helper_Options::get( 'auth_user_data' ) ? get_avatar_url( WC_Helper_Options::get( 'auth_user_data' )['email'], 48 ) : '',
+		);
+		return $settings;
```

Comment:
I would have a line break before the return statement, like we do in `get_connection_url`. The WordPress code style got me into the habit of doing this.

```suggestion

		return $settings;
```